android/src/main/java/org/ergoplatform/android/tokens/TokenDbEntities.kt

```diff
@@ -0,0 +1,94 @@
+package org.ergoplatform.android.tokens
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import org.ergoplatform.persistance.GENUINE_UNKNOWN
+import org.ergoplatform.persistance.TokenInformation
+import org.ergoplatform.persistance.TokenPrice
+import java.math.BigDecimal
+
+@Entity(tableName = "token_price")
+data class TokenPriceDbEntity(
+    @PrimaryKey val tokenId: String,
+    @ColumnInfo(name = "display_name") val displayName: String?,
+    @ColumnInfo(name = "source") val priceSource: String,
+    @ColumnInfo(name = "erg_value") val ergValue: String
```

Comment:
The token erg value can be fractional even compared to nanoergs, and actually is for most tokens. I have experimented with long value/decimals but had problems in deciding how many decimals should actually be used. Since JS developers settled to handle all of these types of values as strings and these are usually our sources, I decided to follow this practice for this application.